docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+name: ft
+
+services:
+  django:
+    container_name: django
+    build: django
+    expose:
+      - 8000
+    init: true
+    networks:
+      - network
+    volumes:
+      - django-src:/app
+      - /static:/static
+      - /media:/media
+    env_file:
+      - .env
+
+  nginx:
+    container_name: nginx
+    build: nginx
+    ports:
+      - 80:80
```

Comment:
Port 80 could already be in use on some computers, for development, using 8080 should be a better option